docs/edge_defaults_manifest.yaml

```diff
@@ -5,17 +5,31 @@ envoy.config.bootstrap.v3.Bootstrap.overload_manager:
     typed_config:
       "@type": type.googleapis.com/envoy.config.resource_monitor.fixed_heap.v2alpha.FixedHeapConfig
       # TODO: Tune for your system.
-      max_heap_size_bytes: 2147483648 # 2 GiB
+      max_heap_size_bytes: 1073741824 # 1 GiB
   actions:
   - name: "envoy.overload_actions.shrink_heap"
     triggers:
     - name: "envoy.resource_monitors.fixed_heap"
       threshold:
-        value: 0.95
+        value: 0.90
   - name: "envoy.overload_actions.stop_accepting_requests"
     triggers:
     - name: "envoy.resource_monitors.fixed_heap"
       threshold:
-        value: 0.98
-
+        value: 0.95
+envoy.config.cluster.v3.Cluster.per_connection_buffer_limit_bytes: 32768 # 32 KiB
+envoy.config.cluster.v3.Cluster.http2_protocol_options:
+  initial_stream_window_size: 65536 # 64 KiB
+  initial_connection_window_size: 1048576 # 1 MiB
 envoy.config.listener.v3.Listener.per_connection_buffer_limit_bytes: 32768 # 32 KiB
+envoy.config.route.v3.RouteAction.idle_timeout: 15s # must be disabled for long-lived and streaming requests
+envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager.common_http_protocol_options:
+  idle_timeout: 3600s # 1 hour
+  headers_with_underscores_action: REJECT_REQUEST
+envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager.http2_protocol_options:
+  max_concurrent_streams: 100
+  initial_stream_window_size: 65536 # 64 KiB
+  initial_connection_window_size: 1048576 # 1 MiB
+envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager.stream_idle_timeout: 300s # 5 mins, must be disabled for long-lived and streaming requests
```

Comment:
I think the comment here needs adjusting. We definitely want this configured for streaming requests.